charts/spire/charts/spire-server/templates/issuer.yaml

```diff
@@ -38,14 +39,14 @@ spec:
   renewBefore: {{ . }}
   {{- end }}
   issuerRef:
-    name: {{ include "spire-server.fullname" $ }}-selfsigned
+    name: {{ $issuerName }}
     kind: Issuer
     group: cert-manager.io
 ---
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: {{ include "spire-server.fullname" $ }}
+  name: {{ include "spire-server.fullname" $ }}-ca
```

Comment:
Considering that this will require coordination between multiple files based on hard-coded '-ca' or '-selfsigned' suffixes, the need to keep each suffix correct between the YAML files is a maintenance burden.

The way to fix this is to define in the _helpers.tpl a new value and the to import that value.  So instead of importing spire-server.fullname, you are importing spire-server.ca-certificate.

That way, if there is ever a decision to change the '-ca' suffix, we can make it once, and guarantee that all the affected YAML files are the same.

Reply:
This is much better, but what about the `-ca` version?  IussuerCA might be a good name for it.

Reply:
Resolving without asking for updates to have a variable for the -ca version.  Internal / external CA handling is not clarified with the necessary support patterns to make such a change trivial.

Reply:
My comment was not resolved, which is why the tests are broken.

Reply:
pushed a commit to fix this.